test/goldens/end2end/_dummy_pkg-1.0.0-null-safety.0.json

```diff
@@ -60,35 +45,36 @@
+        "status": "failed",
+        "summary": "### [x] 0/30 points: code has no errors, warnings, lints, or formatting issues\n\n<details>\n<summary>\nERROR: Target of URI doesn't exist: 'package:retry/retry.dart'.\n</summary>\n\n`lib/_dummy_pkg.dart:1:8`\n\n```\n  ╷\n1 │ import 'package:retry/retry.dart';\n  │        ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  ╵\n```\n\nTo reproduce make sure you are using [pedantic](https://pub.dev/packages/pedantic#using-the-lints) and run `dartanalyzer lib/_dummy_pkg.dart`\n</details>\n<details>\n<summary>\nERROR: The function 'retry' isn't defined.\n</summary>\n\n`lib/_dummy_pkg.dart:5:3`\n\n```\n  ╷\n5 │   retry(() {\n  │   ^^^^^\n  ╵\n```\n\nTo reproduce make sure you are using [pedantic](https://pub.dev/packages/pedantic#using-the-lints) and run `dartanalyzer lib/_dummy_pkg.dart`\n</details>"
       },
       {
         "id": "dependency",
         "title": "Support up-to-date dependencies",
         "grantedPoints": 0,
         "maxPoints": 20,
         "status": "failed",
-        "summary": "### [x] 0/10 points: All of the package dependencies are supported in the latest version\n\n|Package|Constraint|Compatible|Latest|\n|:-|:-|:-|:-|\n|[`retry`]|`^2.0.0`|2.0.0|**3.1.0**|\n\nTo reproduce run `pub outdated --no-dev-dependencies --up-to-date --no-dependency-overrides`.\n\n[`retry`]: https://pub.dev/packages/retry\n\n<details>\n<summary>\nThe constraint `^2.0.0` on retry does not support the stable version `3.0.0`.\n</summary>\n\nTry running `dart pub upgrade --major-versions retry` to update the constraint.\n</details>\n\n### [x] 0/10 points: Package supports latest stable Dart and Flutter SDKs\n\n* Found no Flutter in your PATH. Could not determine the current Flutter version."
+        "summary": "### [x] 0/10 points: All of the package dependencies are supported in the latest version\n\n<details>\n<summary>\nSdk constraint doesn't support current Dart version 2.12.0. Cannot run `pub outdated`.\n</summary>\n\n`pubspec.yaml:8:8`\n\n```\n  ╷\n8 │   sdk: \">=2.12.0-0 <2.12.0\"\n  │        ^^^^^^^^^^^^^^^^^^^^\n  ╵\n```\n\n</details>\n\n### [x] 0/10 points: Package supports latest stable Dart and Flutter SDKs\n\n<details>\n<summary>\nSdk constraint doesn't support current Dart version 2.12.0.\n</summary>\n\n`pubspec.yaml:8:8`\n\n```\n  ╷\n8 │   sdk: \">=2.12.0-0 <2.12.0\"\n  │        ^^^^^^^^^^^^^^^^^^^^\n  ╵\n```\n\nTry widening the upper boundary of the constraint.\n</details>\n* Found no Flutter in your PATH. Could not determine the current Flutter version."
```

Comment:
Seems we should find another version of _dummy_package with a wider sdk-constraint, or another package-version that declares null-safety but uses a legacy library to test on.